Packs/GitLab/pack_metadata.json

```diff
@@ -2,19 +2,20 @@
     "name": "GitLab",
     "description": "Pack for handling gitlab operations",
     "support": "xsoar",
-    "currentVersion": "2.0.0",
+    "currentVersion": "2.1.0",
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
     "created": "2020-04-14T00:00:00Z",
     "categories": [
         "Cloud Services"
     ],
-    "tags": [],
+    "tags": [
+        "Data Source"
+    ],
     "useCases": [],
     "keywords": [],
     "marketplaces": [
-        "marketplacev2",
-        "xsoar"
```

Comment:
why did you remove it?

Reply:
because its only a pack with data source - not supported in XSOAR

Reply:
my mistake